app/src/main/java/org/qosp/notes/ui/tags/TagsViewModel.kt

```diff
@@ -31,6 +49,21 @@ class TagsViewModel @Inject constructor(private val tagRepository: TagRepository
         }
     }
 
+    fun getSortTagsMethod() : String {
+
+        return runBlocking {
+            val sort = async {
+                preferenceRepository
+                    .getAll()
+                    .map { it.sortTagsMethod }
+                    .first()
+                    .name
+            }
+
+            return@runBlocking sort.await()
```

Comment:
Same

Reply:
Done. Carried out in f949e9c4118a72110ecfe55d7de3fbf63caaedbf.